src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
```

Comment:
This interface name should be adjective - `Drawable`

https://howtodoinjava.com/java/basics/java-naming-conventions/